packages/scandipwa/src/component/Popup/Popup.component.js

```diff
@@ -26,13 +26,12 @@ import './Popup.style';
 export class Popup extends Overlay {
     static propTypes = {
         ...Overlay.propTypes,
-        clickOutside: PropTypes.bool,
+        clickOutside: PropTypes.bool.isRequired,
```

Comment:
In ScandiPWA we prefer to add prefixe 'is' to boolean and it will be more sense to rename it to 'isAllowedToClickOutside' or something like that.